### ISSUE TITLE
Certificates are now based on AWS by default

### ISSUE THREAD

**Issue by henrykuijpers** [shinesolutions/puppet-aem-curator#166]
Since commit https://github.com/shinesolutions/puppet-aem-curator/commit/7fd722a5512c456bbaae9a7f060d17f517585db9#diff-fd3213de7fa589296976a86d65cc2496 it seems that using AWS is the only way to get the certificates in place. Unless it is possible to put the certificates somewhere, before calling Curator?

What is the way to go here, since we are not using AWS for our local development machine setup?

**Comment by henrykuijpers** [comment]
```
Error: Evaluation Error: Error while evaluating a Resource Statement, Evaluation Error: Error while evaluating a Function Call, Certificate ARN can only be of types: ( arn:aws:acm | arn:aws:iam | s3: ) (file: /vagrant/aempuppet/modules/aem_curator/manifests/reconfig_pre_aem.pp, line: 236, column: 9) (file: /vagrant/aempuppet/modules/aem_curator/manifests/config_author_primary.pp, line: 119) on node author.localhost.com
```

**Comment by Carnifrex** [comment]
The error message should be updated from "Certificate ARN can only be of types: ( arn:aws:acm | arn:aws:iam | s3: )" to "Certificate ARN can only be of types: ( arn:aws:acm | arn:aws:iam | s3: | http: | https: | file: )"

**Comment by mbloch1986** [comment]
@henrykuijpers I belive @Carnifrex  is right, this more of an issue around the error message. While it actually supports http and file as well.

**Comment by mbloch1986** [comment]
@henrykuijpers & @Carnifrex  Changes are included in release 3.11.0. Therefor I'll close this ticket. please re-open it if anything is still unclear.
